### ISSUE TITLE
UDP Transport

### ISSUE THREAD

**Issue by Xhanti** [davidfowl/BedrockFramework#83]
Hi,

I wanted to know if there are any plans to support a UDP transport in this framework?

Regards

**Comment by jacobslusser** [comment]
I have the same question. It seems like the base abstractions are all geared around connections... which don't lend themselves very well to UDP.

**Comment by davidfowl** [comment]
Right there's no Udp abstraction as yet. It hasn't been designed

**Comment by jacobslusser** [comment]
@davidfowl to make sure I understand, do you agree then that the current abstractions are not correct for UDP? i.e. any UDP implementation would _not_ use `IConnectionFactory`, `ConnectionContext`, etc?

**Comment by davidfowl** [comment]
That's correct. The current abstractions are for streaming, connection based transports.

**Comment by adamradocz** [comment]
UDP protocol would be nice to have. How would you start to implement it? Can you give some guidance?

**Comment by wjkhappy14** [comment]
nice

**Comment by davidfowl** [comment]
I’d need to understand more about the goals. What does the proposed API look like? The current API are for a stream based abstraction, not message based.
